### PR TITLE
Improve display of Storybook as with govuk-react

### PR DESCRIPTION
**Pull request by gavinorland**
Equivalent to https://github.com/UKHomeOffice/govuk-react/pull/374

### CODE REVIEW

File: packages/storybook/src/stories/index.js [hunk 1]
```diff
@@ -1,5 +1,11 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
 import 'normalize.css';
 import './styles.css';
+import readme from '../../../../README.md';
+
+// eslint-disable-next-line react/no-danger
+storiesOf(' Welcome', module).add('to govuk-frederic', () => <div style={{ padding: '10px' }} className="markdown-body" dangerouslySetInnerHTML={{ __html: readme }} />);
```
Comment (Loque-) [inline packages/storybook/src/stories/index.js]: Real shame to use dangerouslySetInnerHTML - could you add a comment to the code as to why this is required plx?

Comment (Loque-) [reply packages/storybook/src/stories/index.js]: I will check this out if I can today to check toooooo

Comment (gavinorland) [reply packages/storybook/src/stories/index.js]: I'm just not sure of another way of doing it. 🤔 If we just output the readme, it's not in markdown but it is rendered as an HTML string. Yep, if you can see a better way then great.

Comment (Loque-) [reply packages/storybook/src/stories/index.js]: Sorted! https://github.com/UKHomeOffice/govuk-react/pull/390/files

**Comment by gavinorland** [comment]
We'll hold off on this and I'll implement Toby's neat solution which uses `withDocs` but hides the component display!

**Comment by stevesims** [comment]
@gavinorland this looks really close - just a small push to get it over the line implementing the changes to avoid the use of `dangerouslySetInnerHTML` - any chance of updating this so we can merge it in?

**Comment by gavinorland** [comment]
@stevesims Yep, I'll do this soon and we can get it in.

**Pull request merged**